commasrl/src/main/java/edu/illinois/cs/cogcomp/comma/datastructures/Sentence.java

```diff
@@ -0,0 +1,254 @@
+/**
+ * This software is released under the University of Illinois/Research and Academic Use License. See
+ * the LICENSE file in the root folder for details. Copyright (c) 2016
+ *
+ * Developed by: The Cognitive Computation Group University of Illinois at Urbana-Champaign
+ * http://cogcomp.cs.illinois.edu/
+ */
+package edu.illinois.cs.cogcomp.comma.datastructures;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import edu.illinois.cs.cogcomp.comma.CommaLabeler;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.PredicateArgumentView;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotationUtilities;
+import edu.illinois.cs.cogcomp.core.utilities.StringUtils;
+
+/**
+ * Used to group all the commas in a sentence. The commas can then be accessed based on positional
+ * relations to each other within the sentence
+ * 
+ * @author navari
+ *
+ */
+public class Sentence implements Serializable {
```

Comment:
This has the same name as a core-utilities class -- should we change it to make it clear where this is used (without checking imports)?